reports.py

```diff
@@ -257,6 +257,26 @@ def predicate(issue):
         predicate=predicate,
         printer=make_console_printer(show_title=True, show_age=True))
 
+
+def incompatible_flag_description(title):
```

Comment:
Nest this function in breaking_changes_1_0 since it's not used anywhere else?

Reply:
I think this is a generally useful function that I want to use elsewhere - will keep for now.